run_pytest.sh

```diff
@@ -10,4 +10,4 @@ SCRIPT_DIR="$(dirname "$SOURCE")"
 cd "$SCRIPT_DIR" || (echo "Could not change to top-level project directory" && exit 1)
 
 # Run pytest
-pytest -c pytest.ini
+pytest -c pytest.ini $1 $2 $3
```

Comment:
wouldn't it be better to pass all parameter with `$@` .
In case there are more parameters needed in the future?

Reply:
you are right :)